src/main/java/com/mongodb/kafka/connect/util/MongoClientHelper.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.kafka.connect.util;
+
+import org.bson.BsonDocument;
+
+import com.mongodb.client.MongoClient;
+
+public final class MongoClientHelper {
+
+  private MongoClientHelper() {}
+
+  public static boolean isAtleastFiveDotZero(final MongoClient mongoClient) {
+    try {
+      return mongoClient
+              .getDatabase("admin")
+              .runCommand(BsonDocument.parse("{hello: 1}"))
+              .get("maxWireVersion", 0)
+          >= 13;
+    } catch (Exception e) {
```

Comment:
[optional]

The code within `try` does not throw checked exceptions, so I propose narrowing the `catch` block to unchecked ones (`RuntimeException`).

Reply:
Done